Formula/crystal-lang.rb

```diff
@@ -1,24 +1,11 @@
 class CrystalLang < Formula
   desc "Fast and statically typed, compiled language with Ruby-like syntax"
   homepage "https://crystal-lang.org/"
-  revision 1
+  url "https://github.com/crystal-lang/crystal/archive/0.20.0.tar.gz"
+  sha256 "97118becc450cdfbafd881f8e98e93d9373d71b4994f49f41dec1eaf081e8894"
   head "https://github.com/crystal-lang/crystal.git"
 
-  stable do
-    url "https://github.com/crystal-lang/crystal/archive/0.19.4.tar.gz"
-    sha256 "e239afa449744e0381823531f6af66407ba1f4b78767bd67a9bb09d9fcc6b9e4"
-
-    # Remove for > 0.19.4
-    # changes already merged upstream to fix compilation with LLVM 3.9
-    # https://github.com/crystal-lang/crystal/pull/3439
-    patch do
-      url "https://github.com/crystal-lang/crystal/commit/13b11d7.patch"
-      sha256 "d981515791c48ae7fce0e906b0eec934fd622987a87f0614b3c91c71b0966b66"
-    end
-  end
-
   bottle do
-    rebuild 1
```

Comment:
Sure! I didn't know what I had to do there so I reverted it to what was there last time I touched this file. Thank you again for promptly reviewing and merging this 🎉